src/tmpfiles/tmpfiles.c

```diff
@@ -2274,16 +2306,22 @@ static int read_config_file(const char *fn, bool ignore_enoent) {
         FOREACH_LINE(line, f, break) {
                 char *l;
                 int k;
+                bool invalid_line = false;
 
                 v++;
 
                 l = strstrip(line);
                 if (IN_SET(*l, 0, '#'))
                         continue;
 
-                k = parse_line(fn, v, l);
-                if (k < 0 && r == 0)
-                        r = k;
+                k = parse_line(fn, v, l, &invalid_line);
+                if (k < 0) {
+                        if (arg_permissive && invalid_line)
+                                ; /* Do nothing */
+                        else if (r == 0)
+                                /* The first error becomes our return value */
+                                r = k;
+                }
         }
```

Comment:
hmm, do we really want an option for this? maybe just make this the normal behaviour?

Reply:
Or maybe define different exit statuses, and report syntax errors and suchlike with error number 100, and then whitelist that exit code in the unit file? That way, users will still get a failure by default, but in the systemd unit we'd happily proceed anyway as long as things are syntax errors

Reply:
after all we do have this beautiful `SuccessExitStatus=` concept in systemd, would be great to actually use it ourselves ;-)